_data/events_gdcr2019/lyon.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "Global day of code retreat - Lyon 2019",
+  "url": "https://www.eventbrite.fr/e/billets-global-day-of-code-retreat-77321997175",
+  "moderators": [
+    {
+      "name": "Software Crafters Lyon",
+      "url": "https://github.com/swcraftlyon"
+    },
+    {
+        "name": "Nastasia"
+    },
+    {
+        "name": "Guillaume"
+    },
+    {
+        "name": "Romain"
+    }
+  ],
+  "sponsors": [],
+  "date": {
+    "start": "2019-11-16T09:00:00.000+01:00",
+    "end": "2019-11-16T17:00:00.000+01:00"
+  },
+  "location": {
+    "city": "LYON",
+    "country": "France",
+    "coordinates": {
```

Comment:
Unless you put in the right coordinates, the event won't show up on our map 🤔

As it will show up on the table though, I'd merge it right now :+1: